<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Судоку</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="verticalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>210</y>
      <width>191</width>
      <height>251</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="layout_buttons">
     <property name="topMargin">
      <number>0</number>
     </property>
     <item>
      <widget class="QPushButton" name="btn_new_game">
       <property name="text">
        <string>Новая игра</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btn_upload_game">
       <property name="text">
        <string>Загрузить игру</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btn_login">
       <property name="text">
        <string>Авторизоваться</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QToolButton" name="btn_settings">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>20</y>
      <width>55</width>
      <height>45</height>
     </rect>
    </property>
    <property name="text">
     <string>...</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>10</y>
      <width>200</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Игра &quot;Судоку&quot;</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>600</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
